### ISSUE TITLE
First page tracking not firing when using $state

### ISSUE THREAD

**Issue by gggritso** [angulartics/angulartics#563]
Heya!

I'm trying to use angulartics with this setup:
```
    .config( function( $analyticsProvider ) {
      $analyticsProvider.firstPageview( true );
      $analyticsProvider.withAutoBase( true );
    })
```

but the initial page load is not being tracked. I believe it's caused by this section:
```
        for (var state in $state.get()) {
          noRoutesOrStates = false;
          break;
        }
```

`$state.get()` returns the parent abstract state with `url` of `^` even for pages with no declared routes. This causes `noRoutesOrStates` to be set to `false` incorrectly and the page tracking code doesn't fire.

**Comment by timelf123** [comment]
PR welcome, thanks for reporting. I don't have time to verify this atm

**Comment by gggritso** [comment]
I'm using `0.2.18`. Working on PR now, but I'm not very confident with ui-router internals.

**Comment by gggritso** [comment]
👏 thank you!

**Comment by timelf123** [comment]
No thank you!

**Comment by gggritso** [comment]
Do you think you’ll be able to cut a new build for npm/bower with this fix in it within the next few days?

**Comment by timelf123** [comment]
Published 1.5.0
